clusterloader2/pkg/test/context.go

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
Can you split this file into two:

- context.go (keep all interface definitions here)
- simplecontext.go (move your implementation of the interface here)

So that it's more easier to understand/maintain.